manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -3963,9 +3963,14 @@ def prune_deleted_volumes(self):
                 try:
                     client.send_request(
                         'volume-destroy', {'name': volume_name})
-                except netapp_api.NaApiError:
+                except netapp_api.NaApiError as e:
                     LOG.error(
                         'Pruning soft-deleted volume %s failed', volume_name)
+                    if e.code == netapp_api.EVOLDEL_NOT_ALLOW_BY_CLONE:
+                        if self.volume_clone_split_status(volume_name) != 100:
+                            self.volume_clone_split_start(volume_name)
+                            LOG.error('Staring clone split for '
```

Comment:
```suggestion
                            LOG.debug('Starting clone split for '
```

We already got an error log above, I think it is sufficient having the information about the split start in log level debug